file-module/src/test/resources/org/mule/weave/io/file/separator/transform.dwl

```diff
@@ -0,0 +1,7 @@
+%dw 2.0
+output application/json
+import * from dw::io::file::FileSystem
+---
+{
+  sep: separator()
```

Comment:
This test is not going to run on windows

Reply:
Addressed in 70af73d